Script/README.md

```diff
@@ -0,0 +1,46 @@
+GrovePi
```

Comment:
@RobertLucian I think we should keep this, but maybe delete or reject the changes below right?

Reply:
I would just close the PR and in case we need it, we can just open it up.